src/main.rs

```diff
@@ -253,6 +253,46 @@ struct BreakpadInfo {
     syms_dir: String,
 }
 
+trait CpuArch {
+    fn cpuarch(&self) -> Arch;
+}
+
+impl CpuArch for mach::header::Header {
+    fn cpuarch(&self) -> Arch {
+        // Copied from symbolic_debug::macho::MachObject::arch.
```

Comment:
Normally someone would complain about this, but I Extremely know the pain of these kinds of aggregations being at the wrong level of abstraction for this stack of libraries...